omskriving av tekstomrade

nutgaard opened the pull request:
Semi-relatert til #651 men ble i all hovedsak gjort pga vi trengte større fleksibilitet fra `Tekstomrade`. 

Eksemplet fra #651 gir en fin pekepinne på hva som endrer seg;

*Input:*
```
1
2
3
4

5
6


7
```

Ga tidligere 7 paragrafer, men basert på feedback fra våre er brukere så forventer de å få tre. Samtidig ønsker der å beholde linjeskift. :P 
Eksemplet ovenfor vil med denne endringen få følgende html-output;
```html
<p class="typo-normal blokk-xs">
1<br>
2<br>
3<br>
4
</p>
<p class="typo-normal blokk-xs">
5<br>
6
</p>
<p class="typo-normal">7</p>
```

Commit:
omskriving av tekstomrade for mer fleksibilitet i bruktområder

Lillebo reviewed (dismissed):
Hm, burde vi utvide dokumentasjonen litt samtidig? Legge til et eksempel som viser hvordan man kan bruke `rules`-propen f.eks.?

Commit:
la til litt mer dokumentasjon av bruken av nytt tekstomrade

Commit:
Merge branch 'master' into redo/tekstomrade

Commit:
La til clean-steg i postinstall for å fjerne gamle lib-filer

Commit:
bumped avhengighet til textparser etter fiks av skrivefeil

Commit:
sletter unna enda flere autogenererte filer

Sletter nÃ¥*.d.ts og lib/ siden disse har en tendens
til Ã¥ henge igjen ved endringer og skape trÃ¸bbel

nutgaard committed:
fikset versjon-nummer for textparser

nutgaard committed:
fikset linting issues i clean-script

Lillebo committed:
Små tweaks på dokumentasjon

Lillebo committed:
typo
